spring-integration-core/src/main/java/org/springframework/integration/aot/CoreRuntimeHints.java

```diff
@@ -166,4 +162,25 @@ private static void registerSpringJdkProxy(ProxyHints proxyHints, Class<?>... pr
 		proxyHints.registerJdkProxy(AopProxyUtils.completeJdkProxyInterfaces(proxiedInterfaces));
 	}
 
+	@SuppressWarnings("removal")
```

Comment:
> ... So, please, bring old entries back with respective `removal` suppression, and add new entries.

_Originally posted by @artembilan in https://github.com/spring-projects/spring-integration/pull/10212#discussion_r2216850652_

I'm not sure this is what you meant.

Reply:
I see what you mean.
Right, let’s mark this method as deprecated to not forget to remove it eventually. Yes , it does not warn because we don’t use API directly. It also looks like these new methods could be static.
On the other hand: Jackson 3 is going to be with us forever, so no need to extract method for it.